_data/simple-icons.json

```diff
@@ -1905,6 +1940,11 @@
             "hex": "FF6B6B",
             "source": "http://www.gotinder.com/press"
         },
+        {
+            "title": "T-mobile",
+            "hex": "ED008C",
```

Comment:
I struggled to find an official source for the T-Mobile brand color, but both [BrandColor](https://brandcolors.net/) and the PNGs I found on the official [media library](https://www.t-mobile.com/news/media-library?fulltext=logo) give the hex value `#E20074` rather then `#ED008C` 🤔

Personally I feel like we should use `#E20074`, agreed? If so, you could possible update the source to the Media Library link as well

Reply:
Can do!